cpp-linter-lib/src/clang_tools/clang_tidy.rs

```diff
@@ -79,6 +79,7 @@ fn parse_tidy_output(
     let note_header = Regex::new(r"^(.+):(\d+):(\d+):\s(\w+):(.*)\[([a-zA-Z\d\-\.]+)\]$").unwrap();
     let mut notification = None;
     let mut result = Vec::new();
+    let cur_dir = current_dir().unwrap();
```

Comment:
Using `.unwrap()` is a bad habit that I should break.

Just posting this to test REST API on a PR review...